<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Current stats</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBoxClass">
          <item>
           <property name="text">
            <string>Archer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Beginner</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bowman</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mage</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pirate (Bucc)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pirate (Sair)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Theif</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Warrior (Fighter)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Warrior (Spearman)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Base hp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxBaseHp">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>2100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Base mp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxBaseMp">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>24600</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Base int</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxBaseInt">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>780</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Current lvl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxBaseLvl">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>148</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Washing options</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QCheckBox" name="checkBoxTargetMp">
              <property name="text">
               <string>Target mp &gt; minimum for target lvl (Primarily for mage)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxTargetMp">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxIncludeHpQuest">
            <property name="text">
             <string>Include hp quest</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="QLabel" name="labelAddIntLvl">
              <property name="text">
               <string>Add int until lvl</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxAddIntLvl">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Wash base int out at lvl</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxWashBaseIntLvl"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Int from gear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxIntGear"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Starting lvl to wear int</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_12"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="checkBoxMpWash">
              <property name="text">
               <string>Mp wash</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="labelMpWashStart">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>start lvl</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxMpWashStart">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="labelMpWashStop">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>stop lvl</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxMpWashStop_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Mw lvl</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxMwLvl">
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Base stat projections</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetBaseStats"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonProcess">
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad_file"/>
    <addaction name="actionSave_file"/>
    <addaction name="actionSave_file_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionSave_file_as">
   <property name="text">
    <string>Save file as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
